src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+class Attendee_List_Display {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Attendee_List_Display

<sub>hash: 66a4f08996e485f4601ac90a24b2fd34</sub>